BUILDING.md

```diff
@@ -43,6 +43,9 @@ need to satisfy this dependencies in your system:
 
 * CentOS/Fedora: `yum install btrfs-progs-devel`
 * Debian/Ubuntu: `apt-get install btrfs-tools`
+	* Debian Buster/Ubuntu 19.10: 
+	    `apt-get install btrfs-progs`
+	    `apt-get install libbtrfs-dev`
```

Comment:
Markdown renders these as one line.
It might worth to combine it in one command:
```suggestion
	    `apt-get install btrfs-progs libbtrfs-dev`
```

Reply:
My bad, should have done that. Will do now.